web/client-ide/nginx/init-notebooks.sh

```diff
@@ -0,0 +1,5 @@
+set -eux
+
```

Comment:
Let's be more explicit w/ our naming here and follow the convention they use. I'd choose `99-init-notebooks.sh` to ensure it runs last given the current naming scheme.